packages/core/src/components/datetime-picker/datetime-picker.tsx

```diff
@@ -34,21 +34,6 @@ export class DatetimePicker
    */
   @Prop() range = true;
 
-  /**
-   * Show hour input
-   */
-  @Prop() showHour = true;
-
-  /**
-   * Show minutes input
-   */
-  @Prop() showMinutes = true;
-
-  /**
-   * Show seconds input
-   */
-  @Prop() showSeconds = true;
-
```

Comment:
Breaking change of API, should be implemented the same workaround like in the time-picker